### PR TITLE
[Pull request Custom layout] Add "Halo infinite" and "Shredders" custom touch layout.

### PR DESCRIPTION
**Pull request by Tak-attack**
I am requesting custom touch layouts for two games I have created.

### CODE REVIEW

File: touch-layouts/1931911684.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+	"name": "Shredders",
+	"default_layout": "custom",
+	"layouts": [
+		"Shredders"
```
Comment (redphx) [inline touch-layouts/1931911684.json]: Please use lowercase for file name

File: touch-layouts/2043073184.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+	"name": "Halo infinite",
```
Comment (redphx) [inline touch-layouts/2043073184.json]: Infinite

File: touch-layouts/1931911684.json (hunk 1 above)
Comment (Tak-attack) [reply touch-layouts/1931911684.json]: Changed file names to all lowercase

**Comment by redphx** [comment]
Thanks for contributing!

**Comment by alanjds** [comment]
@Tak-attack sorry to bother, but what enables the gyro aim in this layout?

Is it possible to enable gyro only when holding a given trigger or bumper?

**Pull request merged**